topup.go

```diff
@@ -53,9 +69,35 @@ type Topup struct {
 	Object                   string              `json:"object"`
 	Source                   *PaymentSource      `json:"source"`
 	StatementDescriptor      string              `json:"statement_descriptor"`
-	Status                   string              `json:"status"`
+	Status                   TopupStatus         `json:"status"`
```

Comment:
breaking change right?

Reply:
No, see above in `terminal_reader.go`: `string -> enum` is non-breaking.